JSON/The Overflow.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Author": "Deaf, kavinkul",
+  "Compatibility": "Compatible",
+  "Contributors": {
+    "Developer": [ "Deaf" ],
+    "Maintainer": [ "Deaf" ],
+    "Manual": [ "Deaf" ]
+  },
```

Comment:
The "Contributors" section needs to be removed since the "Author" section is being used.